### DISCUSSION TITLE
Analytics not tracking all visitors or sources

### DISCUSSION THREAD

**Discussion by datadatafox** [plausible/analytics#2712] [Self-Hosted Support]
Hi, both of my sites I have Plausible enabled for are having two different issues. One of them doesn’t track the sources like it should, at all, and the other seems to only track very little if anything despite a lot of activity. Anyone else having these issues? Unsure of how to contact Plausible to assist.

**Comment by ruslandoga** [comment]
@datadatafox 👋

Can you please describe both issues in a bit more detail:

- how your Plausible instance is set up
- how your tracker script is set up

Also are visitors tracked correctly or not quite? Is the problem only with sources?
